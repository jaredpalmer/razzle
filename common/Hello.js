import React from 'react';

const Hello = () => {
  return (
    <div>
      Sooo Hello...
    </div>
  );
};

export default Hello;
