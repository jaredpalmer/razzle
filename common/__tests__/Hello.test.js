import React from 'react';
import Hello from '../Hello';
import renderer from 'react-test-renderer';

it('renders Hello correctly', () => {
  const tree = renderer.create(<Hello />).toJSON();
  expect(tree).toMatchSnapshot();
});
