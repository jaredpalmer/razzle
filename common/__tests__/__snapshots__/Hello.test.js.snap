// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Hello correctly 1`] = `
<div>
  Sooo Hello...
</div>
`;
